doc/api/http.md

```diff
@@ -1443,6 +1443,31 @@ This event is guaranteed to be passed an instance of the {net.Socket} class,
 a subclass of {stream.Duplex}, unless the user specifies a socket
 type other than {net.Socket}.
 
+### `server.activeConnections()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {net.Socket\[]}
+
+Returns an array of all the sockets connected to the server that have an
+active request.
+
+A request is considered active if it has not fully received by the server and
+it has not expired yet according to `server.headersTimeout` (for the headers) or
+`server.requestTimeout` (for the entire request).
+
+### `server.connections()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {net.Socket\[]}
+
+Returns an array of all the sockets connected to the server.
```

Comment:
When would using this be a good idea?

Reply:
To be honest I don't really have idea.
It's just that we have this informations in the underlying C++ layer and I thought it would be good to expose this to the developer if it actually needs it.